HISTORY.rst

```diff
@@ -2,6 +2,38 @@
 History
 =======
 
+3.39.0 (2021-07-08)
+-------------------
+Critical Changes:
```

Comment:
In retrospect, let's move the Snowfakery stuff from "Critical Changes" into "Changes""